### DISCUSSION TITLE
PL-900 2차에 등록 했으나 "애플리케이션에 액세스하도록 초대함" 이란 메일이 오지않았습니다

### DISCUSSION THREAD

**Discussion by asomi70** [fusiondevkr/pl900#77] [1 Q&A] [answered]
등록 메일은 asomi@naver.com 이며 초대메일이 오지 않아 팀즈 링크도 알지 못합니다. ㅜ.ㅜ 
1차 이수자인데.. 혹 그것때문인가요

**Comment by sesme100** [comment, marked as answer]
안녕하세요 asomi@naver.com으로 PL-900 2차 등록 완료 됨을 확인하였습니다. "애플리케이션에 엑세스하도록 초대함"이라는 메일은 AAD에 등록이 안된 이메일일 경우에만 발송이 됩니다. asomi@naver.com는 퓨전개발자 파워플랫폼 부트캠프에 등록 이력이 있으므로 이미 fusiondev.kr 테넌트 AAD에 등록이 되었습니다. 또한 study@fusiondev.kr 이메일 발신으로 "[퓨전개발자] 기초과정 PL-900 (2차)에 등록하셨습니다!"라는 메일을 받으셨을텐데요. 혹시 못받으셨다면 스팸메일 다시 한번 확인 해주세요!! 혹시라도 못 찾으셨다면 아래 답글에 메세지 남겨주세요! (참고로 참석 팀즈 링크는 11월 15일 이전에 보낼 예정입니다!! )
